examples/training/0-Core Concepts.ipynb

```diff
@@ -0,0 +1,354 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "96c11b32",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Indicate by check mark whether the registrant has filed a report on and attestation to its management’s assessment of the effectiveness of its internal control over financial reporting under Section 404(b) of the Sarbanes-Oxley Act (15 U.S.C. 7262(b)) by the registered public accounting firm that prepared or issued its audit report.  ☐\n",
+      "\n",
+      "\n",
+      "This report contains statements that do not relate to historical or current facts but are “forward-looking” statements. These statements relate to analyses and other information based on forecasts of future results and estimates of amounts not yet determinable. These statements may also relate to future events or trends, our future prospects and proposed new products, services, developments or business strategies, among other things. These statements can generally (although not always) be identified by their use of terms and phrases such as anticipate, appear, believe, could, would, estimate, expect, indicate, intent, may, plan, predict, project, pursue, will continue and other similar terms and phrases, as well as the use of the future tense.\n",
+      "\n",
+      "\n",
+      "Actual results could differ materially from those expressed or implied in our forward-looking statements. Our future financial condition and results of operations, as well as any forward-looking statements, are subject to change and to inherent known and unknown risks and uncertainties. You should not assume at any point in the future that the forward-looking statements in this report are still valid. We do not intend, and undertake no obligation, to update our forward-looking statements to reflect future events or circumstances.\n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from unstructured.documents.elements import NarrativeText\n",
+    "from unstructured.partition.text_type import sentence_count\n",
+    "\n",
+    "for element in elements[:100]:\n",
+    "    if isinstance(element, NarrativeText) and sentence_count(element.text) > 2:\n",
+    "        print(element)\n",
+    "        print(\"\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6b06a398",
+   "metadata": {},
+   "source": [
+    "## Converting to a dictionary <a id=\"dict\"></a>\n",
+    "\n",
+    "The final step in the process for most users is to convert the output to JSON. You can convert document elements by calling the `.to_dict()` method on the `Element` objects:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "5d13fc38",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output = [el.to_dict() for el in elements]"
```

Comment:
This seems like what `convert_to_isd` does. Should we just show them how to use that?

Reply:
Yeah I'll switch this to `convert_to_isd`